src/gt4py/backend/__init__.py

```diff
@@ -14,12 +14,16 @@
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from .concepts import *
+from .base import *
```

Comment:
Agree that `import *` should never be used in a module, but here is used just to import package global symbols from the submodules to the package interface. I also agree that it could be better to do it explicitly but it's not an unusual pattern and it appears in other gt4py subpackages so I will keep as it is in this PR but create an issue to address this globally (#44) in a different commit.